apps/condo/domains/common/constants/import.js

```diff
@@ -1,7 +1,28 @@
+const PROCESSING = 'processing'
+const COMPLETED = 'completed'
+const ERROR = 'error'
+const CANCELLED = 'cancelled'
+
+const IMPORT_STATUS_VALUES = [PROCESSING, COMPLETED, ERROR, CANCELLED]
+
+const EXCEL = 'excel'
```

Comment:
```suggestion
const DOMA_EXCEL = 'doma-excel'
```
To prevent renaming if we will add new formats in future